File: src/main/java/no/nav/eessi/eessifagmodul/EessiFagmodulApplication.java
package no.nav.eessi.eessifagmodul;

import io.prometheus.client.hotspot.DefaultExports;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class EessiFagmodulApplication {

    private static Logger log = LoggerFactory.getLogger(EessiFagmodulApplication.class);

	public static void main(String[] args) {
	    DefaultExports.initialize();
        SpringApplication.run(EessiFagmodulApplication.class, args);
    }
}

File: src/main/kotlin/no/nav/eessi/eessifagmodul/EessiFagmodulApplication.kt
package no.nav.eessi.eessifagmodul

import no.nav.security.spring.oidc.api.EnableOIDCTokenValidation
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@EnableOIDCTokenValidation(ignore = ["org.springframework", "springfox.documentation"])
@SpringBootApplication
class EessiFagmodulApplication

/**
 * under development (Intellij) må hva med under Vm option:
 * -Dspring.profiles.active=local  local run T environment
 *
 */
fun main(args: Array<String>) {
    runApplication<EessiFagmodulApplication>(*args)
}
